online-server/pom.xml

```diff
@@ -38,18 +35,16 @@
             <groupId>com.google.code.gson</groupId>
```

Comment:
There is already a general open issue on this matter, it will be managed in a separate pull request